examples.py

```diff
@@ -0,0 +1,55 @@
+# %%
+from aibrary.resources.aibrary_wrapper import AiBrary
```

Comment:
This should be cleaner.
either 
`import aibrary as aib`
or
`from aibrary import AiBrary`